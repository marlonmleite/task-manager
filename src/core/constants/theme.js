const theme = {
  colors: {
    primary: '#00b7ce',
  },
}

export default theme
