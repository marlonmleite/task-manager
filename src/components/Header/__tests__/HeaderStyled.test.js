import React from 'react'
import { shallow } from 'enzyme'
import theme from 'core/constants/theme'
import HeaderStyled from '../styled'

describe('Header styled', () => {
  it('should render', () => {
    const header = shallow(<HeaderStyled theme={theme} />)

    expect(header).toMatchSnapshot()
  })
})
