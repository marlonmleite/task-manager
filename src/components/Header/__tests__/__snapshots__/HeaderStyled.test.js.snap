// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header styled should render 1`] = `
.c0 {
  background-color: #00b7ce;
  height: 36px;
  padding: 10px;
  box-shadow: 0 0px 12px 0px #0006;
}

<nav
  className="c0"
/>
`;
